docs/dev/env_operators.md

```diff
@@ -0,0 +1,199 @@
+# Operators configuration
```

Comment:
Maybe instead of operators we should say **ACM**? I think it would be more clear. The file should be renamed as well. Keeping the **operators** name could cause confusion that this is for the actual O-RAN O2IMS operator.

Reply:
Yeah, good point. Will update.